packages/react-components/src/molecules/__tests__/TeamCard.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import TeamCard from '../TeamCard';
+
+const member = {
+  id: 'ff0e04ac-4769-44ed-8d3b-245c1bfe17b3',
+  firstName: 'Mason',
+  lastName: 'Carpenter',
+  displayName: 'Birdie Romeo',
+  role: 'VrrPdl',
+};
+const teamCardProps = {
+  id: 'ee98d044-79a7-4028-915d-7f88793e3190',
+  displayName: '!AX54pQ$Jgih7svEaA',
+  applicationNumber: 'P9gr6',
+  projectTitle:
+    'Caczis lu ugez fotsilaz ijmomi uliruti lerohe ji godmiw suuzu imatorok vuk nubozo eveoluf hec sacme sevce wizlec.',
+  projectSummary:
+    'Guj ge te reh hiditzec suw uki cu ziiku tisabe wiwuvev sor jec to gip onrof. Tul edzec zomivbu gotum lakgilzo hip hemgit agzo ew egirub hecguci kistozat hitfankij fiiw muhanti motec eb. Zepo av zimilo jekeker ekrud oti lanidwe peceru faprivsi led sicew sogure ni. Vanbo so fizam wibup hipuh fumuz me agapazcov ucotohfo liwwu ge ki wekezot.',
+  skills: [
+    'Neurological Diseases',
+    'Clinical Neurology',
+    'Adult Neurology',
+    'Neuroimaging',
+    'Neurologic Examination',
+    'Neuroprotection',
+  ],
+  members: [member],
+  lastModifiedDate: '2020-07-31T11:45:14Z',
+};
+
+it('renders the content', () => {
```

Comment:
You can make this just about `renders the team name as the headline`, check that `getByRole('heading')` has the expected text content and has `tagName H2`.
Project summary check is not needed as that pretty declarative - don't need to assert on bits of 'static markup'.
1 Team Member check can go into a test case at the bottom colocated with the other team members test so that there is one for singular and one for plural down there.
Neurological Diseases check is redundant, there are enough tests for the skills.